code/core/src/manager/components/sidebar/Search.tsx

```diff
@@ -90,14 +90,14 @@ const IconWrapper = styled.div(({ theme, onClick }) => ({
   justifyContent: 'center',
 }));
 
-const Input = styled.input(({ theme }) => ({
+const Input = styled.input<{ isMobile: boolean }>(({ theme, isMobile }) => ({
   appearance: 'none',
   height: 28,
   width: '100%',
   padding: 0,
   border: 0,
   background: 'transparent',
-  fontSize: `${theme.typography.size.s1 + 1}px`,
+  fontSize: isMobile ? '16px' : `${theme.typography.size.s1 + 1}px`,
```

Comment:
`1rem` would be even better to correctly account for user zoom preferences, thought that might be better done globally throughout the app in a separate issue.

Reply:
I don't know for sure whether 1rem would satisfy the requirement for the 16px font size behavior, and we also don't use relative font sizes anywhere else. Maybe we'll revisit this at some point but it would be a larger overhaul of the UI.

Reply:
I'm fairly confident it would as we had the same exact a11y violation at Back Market and we switched offending inputs to `1rem` to fix it. I hadn't initially recognised it was the Safari input zoom bug until you commented.

But as you said, `rem` is in use nowhere in the UI so that should be addressed separately.